index.test.ts

```diff
@@ -4,6 +4,8 @@ import { So, Unicode, getSymbolByTitle, getSymbolFromSo } from './index';
 test('get correct unicode symbol', () => {
   const symbol: keyof Unicode = 'GEAR';
   expect(getSymbolByTitle(symbol)).toBe('\u2699');
+  // Symbol with dash
+  expect(getSymbolByTitle('Euro-Currency Sign')).toBe('\u20A0');
```

Comment:
The test case is correct, but it should be capitalized
```suggestion
  expect(getSymbolByTitle('EURO-CURRENCY SIGN')).toBe('\u20A0');
```